test/unit/integrations/mercado_pago/mercado_pago_notification_test.rb

```diff
@@ -0,0 +1,135 @@
+require 'test_helper'
+
+class MercadoPagoNotificationTest < Test::Unit::TestCase
+  include OffsitePayments::Integrations
+
+  def setup
+    @mercado_pago = MercadoPago::Notification.new(custom_ipn_raw_data("approved"))
+  end
+
+  def test_accessors
+    assert @mercado_pago.complete?
+    assert_equal "Completed", @mercado_pago.status
+    assert_equal 1234567, @mercado_pago.transaction_id
+    assert_equal "RTV-25640", @mercado_pago.item_id
+    assert_equal 100, @mercado_pago.gross
+    assert_equal "ARS", @mercado_pago.currency
+    assert_equal "2014-01-30T18:14:24Z", @mercado_pago.received_at
+    assert_equal "payer@example.com", @mercado_pago.payer_email
+    assert_equal "collector@example.com", @mercado_pago.receiver_email
+    assert_equal @mercado_pago.test, true
+  end
+
+  def test_pending
+    @mercado_pago = MercadoPago::Notification.new(custom_ipn_raw_data("pending"))
+    refute @mercado_pago.complete?
+    assert_equal "Pending", @mercado_pago.status
+  end
+
+  def test_rejected
+    @mercado_pago = MercadoPago::Notification.new(custom_ipn_raw_data("rejected"))
+    refute @mercado_pago.complete?
+    assert_equal "Failed", @mercado_pago.status
+  end
+
+  def test_compositions
+    assert_equal Money.new(10000, 'ARS'), @mercado_pago.amount
+  end
+
+  def test_respond_to_acknowledge
+    assert @mercado_pago.respond_to?(:acknowledge)
+  end
+
```

Comment:
I'd like to see a test where `get_access_token` fails with 404 or something (login/password) are wrong.